docs/source/cloud-setup/cloud-permissions/gcp.rst

```diff
@@ -134,17 +134,26 @@ User
     compute.firewalls.list
     compute.firewalls.update
 
-8. Click **Create** to create the role.
-9. Go back to the "IAM" tab and click on **GRANT ACCESS**.
-10. Fill in the email address of the user in the “Add principals” section, and select ``minimal-skypilot-role`` in the “Assign roles” section. Click **Save**.
+8. **Optional**: If the user needs to use custom machine images with ``sky launch --image-id``, you can additionally add the following permissions:
+
+.. code-block:: text
+    
+    compute.disks.get
```

Comment:
Where is this API being called? Correct me if I'm wrong, but I only saw `instances.get` and `disks.resize` in your code? ditto for `compute.images.get` and `compute.images.useReadOnly`

Reply:
The guidance provided in this documentation specifies the minimum permissions required for custom machine images, rather than disk resizing. Therefore we also need ```compute.images.get``` (retrieve information about a specific image) and ```compute.images.useReadOnly``` (allows a user to use a specific image to create instances).